src/sentry/integrations/slack/notifications.py

```diff
@@ -67,21 +71,48 @@ def _notify_recipient(
         # Make a local copy to which we can append.
         local_attachments = copy(attachments)
 
-        # Add optional billing related attachment.
-        additional_attachment = get_additional_attachment(integration, notification.organization)
-        if additional_attachment:
-            local_attachments.append(additional_attachment)
-
-        # unfurl_links and unfurl_media are needed to preserve the intended message format
-        # and prevent the app from replying with help text to the unfurl
-        payload = {
-            "channel": channel,
-            "link_names": 1,
-            "unfurl_links": False,
-            "unfurl_media": False,
-            "text": notification.get_notification_title(ExternalProviders.SLACK, shared_context),
-            "attachments": json.dumps(local_attachments),
-        }
+        text = notification.get_notification_title(ExternalProviders.SLACK, shared_context)
+
+        if features.has("organizations:slack-block-kit", notification.organization):
+            # TODO: enable billing attachments for block kit
+            blocks = []
+            if text:
+                # equivalent of "text" field in legacy attachment system ("text" field does not
+                # show up in notification for block system but is instead used as the fallback text)
+                # ie. "Issue marked as regression", "New comment by <user>"
+                blocks.append(BlockSlackMessageBuilder.get_markdown_block(text))
```

Comment:
Is this working as the preview text for you? According to https://api.slack.com/reference/messaging/payload it should be just `["text"] = "some text"` on the same level as blocks, but I was having intermittent problems with it.

Reply:
so the fallback text is passed in the payload a few lines below (as `text`), this part is part of the notification you see in slack

<img width="315" alt="image" src="https://github.com/getsentry/sentry/assets/45607721/073dfbf3-b162-47d8-9758-ab62e94d0379">

for example, "New comment by isabella@testing.com"
with legacy attachments, you could just use `text` in the payload and that would be used as both preview text and as the first part of the notification in slack itself.
with blocks, whatever you pass in as `text` in the payload will be used as fallback but won't actually be rendered in the notification, hence this line. 

i can try to make the comment more clear tmrw morning!